x-pack/apm-server/sampling/groups.go

```diff
@@ -224,7 +224,7 @@ func (g *traceGroup) finalizeSampledTraces(traceIDs []string, ingestRateDecayFac
 
 	// Resize the reservoir, so that it can hold the desired fraction of
 	// the observed ingest rate.
-	newReservoirSize := int(math.Round(g.samplingFraction * g.ingestRate))
+	newReservoirSize := int(math.Ceil(g.samplingFraction * g.ingestRate))
```

Comment:
[For reviewer] I don't think this is needed for the fix to work, and that we have a min size. But might be good to be consistent and use math.Ceil in all samplingFraction multiplication.